python/lsst/pipe/tasks/makeDiscreteSkyMap.py

```diff
@@ -62,21 +62,30 @@ def setDefaults(self):
         self.skyMap.tractOverlap = 0.0
 
 class MakeDiscreteSkyMapRunner(pipeBase.TaskRunner):
-    """Want to run on all the dataRefs at once, not one at a time."""
+    """Run a task with all dataRefs at once, rather than one dataRef at a time.
+
+    Call the run method of the task using two positional arguments:
+    - butler: data butler
+    - dataRefList: list of all dataRefs,
+    """
     @staticmethod
     def getTargetList(parsedCmd):
         return [(parsedCmd.butler, parsedCmd.id.refList)]
 
-    def precall(self, parsedCmd):
-        # We overload to disable writing/checking of schemas and configs.
-        # There's only one SkyMap per rerun anyway, so the config is redundant,
-        # and checking it means we can't overwrite or append to one once we've
-        # written it.
-        return True
-
     def __call__(self, args):
+        """
+        @param args     Arguments for Task.run()
+
+        @return:
+        - None if doReturnResults false
```

Comment:
Maybe say "`self.doReturnResults`"; took me a moment to remember where that setting comes from.